pkg/kf/commands/utils/prefix_writer.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+)
+
+type prefixWriter struct {
+	nw newlineWriter
+}
+
+func NewPrefixWriter(prefix string, dest io.Writer) io.Writer {
+	return &prefixWriter{
+		nw: newlineWriter{
+			dest: func(data []byte) (int, error) {
+				n, err := fmt.Fprintf(dest, "%s%s\n", prefix, data)
+				if err != nil {
+					return 0, err
+				}
+				return n - len(prefix), nil
+			},
+		},
+	}
+}
+
+func (w *prefixWriter) Write(data []byte) (int, error) {
+	return w.nw.Write(data)
+}
+
+type newlineWriter struct {
+	dest func([]byte) (int, error)
```

Comment:
I think this should be an io.Writer

Reply:
I'm not sure I agree. This is an internal type. If `dest` were a `io.Writer`, then I would have to add quite a bit of boiler plate to have the 2 different functions using it implement the interface.